app/controllers/api/v2/compute_resources_controller.rb

```diff
@@ -30,7 +30,7 @@ def show
         param :compute_resource, Hash, :required => true, :action_aware => true do
           param :name, String, :required => true
           param :provider, String, :desc => N_("Providers include %{providers}") # values are defined in apipie initializer
-          param :url, String, :desc => N_("URL for Libvirt, oVirt, and OpenStack")
+          param :url, String, :desc => N_("URL for Docker, Libvirt, oVirt, OpenStack and Rackspace")
```

Comment:
Sorry - Docker is not a compute resource provided by Foreman, but by https://github.com/theforeman/foreman-docker/ , it should be added there